pyproject.toml

```diff
@@ -7,48 +7,52 @@ build-backend = "flit_core.buildapi"
 [project]
 name = "sonar-space"
 readme = "README.md"
-authors = [{ name = "Meta AI Research" }]
-requires-python = ">=3.8"
+authors = [{name = "Meta AI Research"}]
+requires-python = ">=3.10"
 dynamic = ["version", "description"]
-keywords = [
-  "sentence embeddings",
-  "sentence representation",
-  "sentence encoder",
-  "sonar models",
-  "speech2speech",
-  "text2text",
-  "speech2text",
-  "text2speech",
-  "multi-modal models",
-  "multi-language models",
-]
+keywords = ["sentence embeddings", "sentence representation", "sentence encoder",
+            "sonar models", "speech2speech", "text2text", "speech2text",
+            "text2speech", "multi-modal models", "multi-language models"]
+
 
 # zip_safe = false
-classifiers = [
-  "Topic :: Scientific/Engineering",
-  "Development Status :: 4 - Beta",
+classifiers=[
+    "Topic :: Scientific/Engineering",
+    "Development Status :: 4 - Beta",
 ]
 
 dependencies = [
-#   "fairseq2>=0.3.0rc1", # installed with the cpu deps, if you want a gpu build, you need to install it manually
+  # fairseq2 is installed with the cpu deps; if you want a gpu build, you need to install it manually.
+  # see https://github.com/facebookresearch/fairseq2
+  "fairseq2>=0.4.0",
```

Comment:
```suggestion
  "fairseq2~0.4.",
```

Reply:
I am looking at the version specification for compatible releases: https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release
We probably want any version of `fairseq2` that is not older than `0.4.0`, but is older than `0.5` (because the 0.5 might include something incompatible again). The right way to specify this seems to be `fairseq2~=0.4.0`.